resources/scripts/components/server/ServerDetailsBlock.tsx

```diff
@@ -74,6 +74,7 @@ const ServerDetailsBlock = () => {
 
     const diskLimit = limits.disk ? megabytesToHuman(limits.disk) : 'Unlimited';
     const memoryLimit = limits.memory ? megabytesToHuman(limits.memory) : 'Unlimited';
+    const cpuLimit = limits.cpu !== 0 ? limits.cpu + '%' : 'Unlimited';
```

Comment:
Maybe do this one like the others above? This would error if `limits.cpu` is undefined.

Reply:
That was some kind of nosleep brainfart. Thanks for pointing it out.